src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
Ah yes - now I recall why I gave up when trying to do this. You need to include all the syntactic plugins manually when you remove transformation presets.